lua/nvim-tree/view.lua

```diff
@@ -152,9 +151,10 @@ end
 
 local function open_window()
   if M.View.float.enable then
-    vim.api.nvim_open_win(0, true, open_win_config())
+    local winid = vim.api.nvim_open_win(M.get_bufnr(), false, open_win_config())
+    vim.api.nvim_set_current_win(winid)
```

Comment:
When set second param(should enter the window) of  `nvim_open_win` to `true`, telescope immediately closes it.` Workaround with `nvim_set_current_win` works fine, Should i leave comment here?